editor/editor_help.cpp

```diff
@@ -2395,6 +2396,24 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 
 		if (brk_pos > pos) {
 			String text = bbcode.substr(pos, brk_pos - pos);
+			if (codeblock_tag && code_indent_type == 0) {
```

Comment:
Could use a comment on where the 0 comes from.

Reply:
I'd rather modify this part here to use a bool.

In a prior version of this PR I also handled spaces and adjusted the padding depending on `text_editor/behavior/indent/size`, but in practice it looked very bad.